Notifier-service/src/services/streams/conferenceStream.ts

```diff
@@ -39,16 +43,72 @@ export class ConferenceStreamMongo extends ConferenceStream {
             })
         })
             .pipe(
-                share()
+                filter()
             )
+        this.insertstreamObs$ = Observable.create((observer: Observer<any>) => {
+            changeStream.then(stream => {
+                stream.on("insert", (data:Object) => {
```

Comment:
* Is the event legit ? i.e does "insert" event exist, because I remember reading u have to create the watch based on filtering the events. Do check that again.